scripts/newinstall.sh

```diff
@@ -187,11 +187,12 @@ fi
 
 if true; then
 	# Check the version by running a small Python program (taken from the Python EUPS package)
+    # XXX should not assume that python is in the default path
```

Comment:
I did a force push to fix the tabs before you commented... not sure why github isn't rendering it.